api/bolt/endpoint/endpoint.go

```diff
@@ -2,7 +2,7 @@ package endpoint
 
 import (
 	"github.com/boltdb/bolt"
-	"github.com/portainer/portainer/api"
+	portainer "github.com/portainer/portainer/api"
```

Comment:
Look at your IDE to fix your auto-import feature, this import update is not required.